libs/api.lua

```diff
@@ -192,8 +194,12 @@ return function (db, prefix)
         search = prefix .. "/search/{query}",
       }
     end,
-    "^/packages/([^/]+)/(.+)/v([^/]+)%.zip$", function (author, name, version)
-
+    "^/packages/([^/]+)/(.+)/([^/]+)%.zip$", function (author, name, version)
+      if version == "latest" then
+        version = (db.offlineMatch or db.match)(author, name)
+      else
+        version = version:sub(1,1) == "v" and version:sub(2) or version
```

Comment:
since we already have an `else` block here, should probably use:

```lua
elseif version:sub(1,1) == "v" then
  version = version:sub(2)
```
... so as to not assign version to itself in the common case.